### PR TITLE
Add validation option to vault persist all

### PR DESCRIPTION
**Pull request by popovm**
* In case of invalid records we might still want to persist them. Some records could be invalid, but we still need to encrypt their information.
* Drop support of ruby `2.2`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Vault Rails Changelog
+## 0.7.6 (February 27, 2019)
```
Comment (bliof) [inline CHANGELOG.md]: 0.8 ?

Comment (popovm) [reply CHANGELOG.md]: The changes from this pull request are not that critical, I think 🤔

Comment (bliof) [reply CHANGELOG.md]: > * Drop support of ruby `2.2` :trollface: 

I am for making it 1.0.0 🎉

Comment (bliof) [reply CHANGELOG.md]: Merge it like this. We talked that 0.6 - rails 4 and 0.7 - rails 5 are maintained and we have to merge them and then go with 1.0.0 :)

Comment (popovm) [reply CHANGELOG.md]: 🤝

**Pull request merged**